lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -159,25 +159,25 @@ def run!(verify_result: true, allow_protected_ns: false, prune: true)
         MSG
         @logger.summary.add_paragraph(ColorizedString.new(warning).yellow)
       end
-      ::StatsD.event("Deployment of #{@namespace} succeeded",
+      StatsD.event("Deployment of #{@namespace} succeeded",
         "Successfully deployed all #{@namespace} resources to #{@context}",
         alert_type: "success", tags: statsd_tags << "status:success")
-      ::StatsD.measure('all_resources.duration', StatsD.duration(start), tags: statsd_tags << "status:success")
+      StatsD.measure('all_resources.duration', StatsD.duration(start), tags: statsd_tags << "status:success")
```

Comment:
The Policial complaint is already addressed in this PR: https://github.com/Shopify/kubernetes-deploy/pull/386. That's going to be a conflict, so I'd say let's just make those changes in this PR and close that one.

Reply:
🤦‍♂️